modules/javafx.graphics/src/main/java/javafx/scene/text/TabStopPolicy.java

```diff
@@ -56,16 +56,17 @@ public final ObservableList<TabStop> tabStops() {
     }
 
     /**
-     * Specifies the default tab stop interval (beyond the last tab stop provided by {@code #tabStops()}),
-     * as a fixed repeating distance (in pixels) to the next tab stop computed at regular intervals
-     * relative to the leading edge of the {@code TextFlow} node.
+     * Specifies the default tab stop interval for tabs beyond the last stop provided
+     * by {@code #tabStops()}.  This is a fixed repeating distance (in pixels) to the
```

Comment:
Change this to a `@link` or remove the `#` and `()` since they show up in the docs.